tests/PSClickWrap.test.js

```diff
@@ -0,0 +1,163 @@
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" confirmationEmail />);
+    expect(_ps.mock.calls[1][2].confirmation_email).toBe(true);
+  });
+
+  it('leaves confirmationEmail as undefined if not passed as prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[1][2].confirmation_email).toBeUndefined();
+  });
+
+  it('sets disableSending to true if passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" disableSending />);
+    expect(_ps.mock.calls[0][2].disable_sending).toBe(true);
+  });
+
+  it('defaults disableSending to false if not passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[0][2].disable_sending).toBe(false);
+  });
+
+  it('defaults displayAll to true if not passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[1][2].display_all).toBe(true);
+  });
+
+  it('sets displayAll to value passed as prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" displayAll={false} />);
+    expect(_ps.mock.calls[1][2].display_all).toBe(false);
+  });
+
+  it('sets auto_run to true by default if not passing displayImmediately to false', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[1][2].auto_run).toBe(true);
+  });
+
+  it('sets auto_run to false if passing displayImmediately=false as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" displayImmediately={false} />);
+    expect(_ps.mock.calls[1][2].auto_run).toBe(false);
+  });
+
+  it('sets dynamic to true in create call if passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" dynamic renderData={{}} />);
+    expect(_ps.mock.calls[0][2].dynamic).toBe(true);
+  });
+
+  it('defaults to dynamic as false if not passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
+    expect(_ps.mock.calls[0][2].dynamic).toBe(false);
+  });
+
+  it('defaults force_scroll to undefined if not passed as a prop', () => {
+    mount(<PSClickWrap accessId="29ea80d9-d386-4cfd-a280-505e802ee732" groupKey="example-clickwrap" signerId="test@abc.com" />);
```

Comment:
The example accessId is used many times throughout the tests, may be worthwhile to make this a global constant so it could be replaced with a different access token easily in the future? Maybe something similar to how the groupKey was mananged when initializing the PSBrowseWrap (`this.targetSelector = 'psbw-${groupKey}';`)

Reply:
Im actually going to be moving this out to an environment variable so it is not committed, ill add steps for adding this back in the docs